text_parser.go

```diff
@@ -435,6 +456,28 @@ func (p *textPlistParser) parseGNUStepValue() cfValue {
 	return nil
 }
 
+// the <[ have already been consumed
+func (p *textPlistParser) parseGNUStepBase64() cfData {
+	p.scanUntil(']')
+	v := p.emit()
+
+	if p.next() != ']' {
+		p.error("invalid GNUStep base64 data (expected ']')")
+	}
+	
+	if p.next() != '>' {
+		p.error("invalid GNUStep base64 data (expected '>')")
+	}
+
+	// Emulate NSDataBase64DecodingIgnoreUnknownCharacters
+	v_filt = Strings.Map(keepCharValidBase64, v)
+	data, err := base64.StdEncoding.DecodeString(v_filt)
```

Comment:
I like this -- using a filter to Keep instead of replacing non-matching characters. Is there an opportunity for consolidation with what the xml parser does for base64 data? If so, it could live in a common place.

https://github.com/DHowett/go-plist/blob/master/xml_parser.go#L136-L140

If gnustep is less strict than apple-xml, I'm fine duplicating the logic for now.

Reply:
my only concern here: does `Strings.Map` (should probably be `strings.Map`?) exist on Go 1.2? plist currently targets 1.2, but it does ship some shims for differences in different versions